### ISSUE TITLE
JW Player Key/Secret test verification

### ISSUE THREAD

**Issue by rveitch** [openfcci/fcc-northland-outdoors-radio#16]
Add functionality to test and verify the JW API key/secret set in the plugin settings page.

**Two ideas for visual confirmation:**
1. A test button, -or-
2. An admin notice is displayed when the options page is saved

**Validation Abstraction:**
- Send test call to JW Player API → return success or fail → display corresponding success/fail message
  - If necessary, add a value to the options table to mark settings as valid

### Reference:
- [JW API Status Call - Documentation](https://developer.jwplayer.com/jw-platform/reference/v1/methods/status.html)
- https://support.jwplayer.com/customer/en/portal/articles/1489902-jw-platform-api-reference

**Comment by rveitch** [comment]
```php
/**
 * Verify the API key and secret that the user has given, by making a call to
 * the API.
 *
 * If the credentials are invalid, return false.
 *
 * If the API call failed, return NULL.
 */
function fcc_jw_account_verify_api_key_secret() {
	$botr_api = new BotrAPI( get_option('options_jw_platform_api_key'), get_option('options_jw_platform_api_secret') ); // Instantiate the API.
	$response = $botr_api->call("/accounts/list"); // Call the API
		if ( isset( $response ) && isset( $response['status'] ) ) {
		if ( 'ok' === $response['status'] ) {
			return true;
		}
		return false;
	}
	return null;
}
```
